packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -202,6 +209,21 @@ export class FeatureDevController {
         }
     }
 
+    private async processChatItemFeedbackMessage(message: any) {
```

Comment:
Is there no type for message? If yes, let's use it. If not, why?

Reply:
No, All messages are `any` in the controller, I will add an action item in the backlog.